app/views/sessions/new.html.erb

```diff
@@ -14,7 +14,7 @@
         <%= f.password_field :password, class: "form-control", placeholder: "Enter your password" %>
       </div>
     </div>
-    <%= hidden_field_tag "account_verified", @account_verified  %>
+    # <%= hidden_field_tag "account_verified", @account_verified  %>
```

Comment:
Here you have a typo you left a "#" symbol and it appears on the log in screen

Reply:
thanks fausitno. I deleted the comment. FIXED